src/test/java/uk/gov/pay/adminusers/resources/UserResourcePatchTest.java

```diff
@@ -57,6 +57,23 @@ public void shouldUpdateTelephoneNumber_whenPatchAttempt() throws Exception {
     }
 
     @Test
+    public void shouldUpdateFeatures_whenPatchAttempt() throws Exception {
+
+        String newFeatures = "SUPER_FEATURE_1, SECRET_SQUIRREL_FEATURE";
+        JsonNode payload = new ObjectMapper().valueToTree(ImmutableMap.of("op", "replace", "path", "features", "value",  newFeatures));
```

Comment:
Really picky but there is an extra space before `newFeatures`

Reply:
done :-)